apps/test-suite/tests/NewNotifications.js

```diff
@@ -1179,6 +1217,69 @@ export async function test(t) {
         10000
       );
     });
+
+    onlyInteractiveDescribe(
+      'triggers a repeating daily notification. only first scheduled event is awaited and verified.',
+      () => {
+        let timesSpyHasBeenCalled = 0;
+        const identifier = 'test-scheduled-notification';
+        const notification = {
+          title: 'Scheduled notification',
+          data: { key: 'value' },
+          badge: 2,
+          vibrate: [100, 100, 100, 100, 100, 100],
+          color: '#FF0000',
+        };
+
+        t.beforeEach(async () => {
+          Notifications.setNotificationHandler({
+            handleNotification: async () => {
+              timesSpyHasBeenCalled += 1;
+              return {
+                shouldShowAlert: false,
+              };
+            },
+          });
+        });
+
+        t.afterEach(() => {
+          Notifications.setNotificationHandler(null);
+          Notifications.cancelAllScheduledNotificationsAsync();
+        });
+
+        t.it(
+          'triggers a repeating daily notification. only first event is verified.',
+          async () => {
+            const dateIn1Mimutes = new Date(
+              new Date().getTime() + (Platform.OS === 'ios' ? 120000 : 60000)
+            );
+            await Notifications.scheduleNotificationAsync({
+              identifier,
+              content: notification,
+              trigger: {
+                hour: dateIn1Mimutes.getHours(),
+                minute: dateIn1Mimutes.getMinutes(),
+              },
+            });
+            const result = await Notifications.getAllScheduledNotificationsAsync();
+            const scheduledTime = new Date();
+            scheduledTime.setSeconds(0);
+            scheduledTime.setMilliseconds(0);
+            if (Platform.OS === 'android') {
+              scheduledTime.setMinutes(result[0].trigger.minute);
+            } else if (Platform.OS === 'ios') {
+              scheduledTime.setMinutes(result[0].trigger.dateComponents.minute);
+            } else {
+              throw new Error('Test does not support platfrom');
+            }
+            const miliSecondsToWait = scheduledTime - new Date().getTime() + 2000;
+            await waitFor(miliSecondsToWait);
```

Comment:
I'm looking up the scheduled time here in order to reduce the wait time here to a minimum.